AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -814,6 +814,15 @@ protected Void doInBackground(Void... params) {
               Constants.loggingEnabled = true;
             }
 
+            JSONArray enabledCounters = response.optJSONArray(
+                    Constants.Keys.ENABLED_COUNTERS);
+            if (enabledCounters != null) {
+              for (int i=0; i < enabledCounters.length(); i++) {
+                String name = enabledCounters.getString(i);
+                Log.enableCounter(name);
```

Comment:
Why not
```Java
Set counterSet = new HashSet<>(Arrays.asList(enabledCounters));
Log.setEnabledCounters(counterSet);
```
Or make `setEnabledCounters` something that takes an array? Simplifies some logic.